src/main/java/io/leonis/subra/game/rule/BallOutOfBoundsRule.java

```diff
@@ -1,7 +1,7 @@
 package io.leonis.subra.game.rule;
 
-import java.util.Set;
 import io.leonis.subra.game.data.*;
+import java.util.Set;
```

Comment:
did you just alphabetically order the imports :') ?

Reply:
![trell](https://user-images.githubusercontent.com/406658/33939711-77ab7082-e00c-11e7-82e5-3fbf00869218.jpg)